src/agents/gmail_agent/config.yml

```
@@ -0,0 +1,31 @@
+name: Gmail Agent
+description: Gmail Agent
+model: google-gla:gemini-2.5-flash
+host: localhost
+port: 10021
+
+system_prompt: |
+  You are **Gmail Agent Pro**, a highly efficient and intelligent AI assistant dedicated to managing the "connor@connorbell.dev" inbox. Your role is to seamlessly handle email-related tasks with precision and an understanding of the user's communication style.
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)